### ISSUE TITLE
How to  test my own model ?

### ISSUE THREAD

**Issue by xiaoping2015** [eric612/MobileNet-YOLO#271]
Hi, I use the train_yolov3_lite.sh to train my own data for object detection. Now, I have the following questions.
First, how to select the pre-training model ?  Mobilenet_yolov3_lite_coco.caffemodel  or mobilenet_yolov3_deploy_iter_78000.caffemodel ?
 I use the mobilenet_yolov3_lite_coco.caffemodel as the pre-trainin model. But, when I test the .caffemode by  /MobileNet-YOLO/examples/yolo/detect.py , it  report the following error :F0922 10:07:17.432552 18394 net.cpp:753] Check failed: target_blobs.size() == source_layer.blobs_size() (2 vs. 1) Incompatible number of blobs for layer conv0
*** Check failure stack trace: ***
Aborted (core dumped)

Second,  how to test the generated file( .caffemodel)? 
I use the  /MobileNet-YOLO/examples/yolo/detect.py file to test the .caffemode, but I don't know how to choose the "    parser.add_argument('--model_def', default='models/yolov3/mobilenet_yolov3_lite_deploy.prototxt')".

Third,  how to solve the problem of slow training speed? 
Now, Iterate every 3 minutes.

Finally, how to solve the problem of non-convergence?
The log is described as follows:
I0922 10:05:56.967492 24358 solver.cpp:253] Iteration 361 (0.00539362 iter/s, 185.404s/1 iters), loss = 135.502
I0922 10:05:56.967684 24358 solver.cpp:272]     Train net output #0: det_loss1 = 1.59122 (* 1 = 1.59122 loss)
I0922 10:05:56.967690 24358 solver.cpp:272]     Train net output #1: det_loss2 = 130.485 (* 1 = 130.485 loss)
I0922 10:05:56.967696 24358 sgd_solver.cpp:121] Iteration 361, lr = 0.0005
I0922 10:05:56.969641 24358 solver.cpp:764] Snapshotting to binary proto file /MobileNet-YOLO/data/VOCdevkit/snapshot/mobilenet_yolov3_lite_solver_iter_362.caffemodel
I0922 10:05:57.025666 24358 sgd_solver.cpp:293] Snapshotting solver state to binary proto file /MobileNet-YOLO/data/VOCdevkit/snapshot/mobilenet_yolov3_lite_solver_iter_362.solverstate
I0922 10:07:32.978587 24358 yolov3_layer.cpp:764] noobj: 0.0010457 obj: 0.276943 iou: 0.684684 cat: 0.94998 recall: 0.959479 recall75: 0.345491 count: 9
I0922 10:07:33.604300 24358 yolov3_layer.cpp:764] noobj: 0.0157246 obj: 0.656418 iou: 0.174302 cat: 0.949975 recall: 0.132913 recall75: 0.020589 count: 646
I0922 10:08:38.741518 24358 solver.cpp:253] Iteration 362 (0.00618147 iter/s, 161.774s/1 iters), loss = 134.816
I0922 10:08:38.741643 24358 solver.cpp:272]     Train net output #0: det_loss1 = 0.395159 (* 1 = 0.395159 loss)
I0922 10:08:38.741652 24358 solver.cpp:272]     Train net output #1: det_loss2 = 177.737 (* 1 = 177.737 loss)

Help me, please.
I am waiting for your reply.

**Comment by xiaoping2015** [comment]
@eric612

**Comment by GeneralSong** [comment]
1.现在也在研究这个yolov3-lite开始训练,没有预训练模型，您能发我一下吗?
mobilenet_yolov3_lite_coco.caffemodel 这个模型下载地址能发我一下吗? 谢谢

**Comment by eric612** [comment]
我放在 #245

**Comment by GeneralSong** [comment]
谢谢了

**Comment by GeneralSong** [comment]
> 我放在 #245

谢谢，找到了
